src/ReactWrapper.jsx

```diff
@@ -362,9 +362,7 @@ class ReactWrapper {
    * @returns {Boolean}
    */
   containsAllMatchingElements(nodes) {
-    const invertedEquals = (n1, n2) => instEqual(n2, n1, (a, b) => a <= b);
-    const predicate = other => containsChildrenSubArray(invertedEquals, other, nodes);
-    return findWhereUnwrapped(this, predicate).length > 0;
+    return Array.isArray(nodes) && nodes.every(node => this.containsMatchingElement(node));
```

Comment:
i think perhaps we should be throwing a `TypeError` here if `nodes` is not an array, rather than just silently returning false.